### PR TITLE
Minor tidy up of ScanProgressDialog class

### PR DESCRIPTION
**Pull request by thc202**
- Remove commented statement;
 - Remove unnecessary instance variable initialisations (already
 initialised by default to null/false);
 - Remove instance variable "site" needed only once to set the title
 (now done in the constructor);
 - Document the constructor;
 - Remove unnecessary JavaDoc comments (incomplete ones from overridden
 methods);
 - Rename the method showProgress to updateProgress (and update its JavaDoc);
 - Change ScanProgressActionListener to:
  - Extend MouseAdapter instead of implementing the interfaces directly,
  as it doesn't require to override unused methods;
  - Use the methods JTable.columnAtPoint/rowAtPoint instead of
  calculating the position of the column/row "manually".

### CODE REVIEW

File: src/org/zaproxy/zap/extension/ascan/ScanProgressDialog.java [hunk 1]
```diff
@@ -360,11 +357,8 @@ private JTable getMainPanel() {
     }
     
     /**
-     * Update the current Scan progress and
-     * prepare actions and scan summary
+     * Updates the current Scan progress and prepare actions and scan summary.
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/ascan/ScanProgressDialog.java]: And and, seems a little confusing.

**Comment by kingthorin** [comment]
One minor thing, otherwise looks good.

File: src/org/zaproxy/zap/extension/ascan/ScanProgressDialog.java (hunk 1 above)
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascan/ScanProgressDialog.java]: Renamed the method (to `updateProgress`) and rewrote its JavaDoc.

**Comment by kingthorin** [comment]
Looks goodLeave a comment

**Pull request merged**